fastlane/lib/fastlane/actions/opt_out_crash_reporting.rb

```diff
@@ -2,15 +2,15 @@ module Fastlane
   module Actions
     class OptOutCrashReportingAction < Action
       def self.run(params)
-        UI.message("fastlane doesn't have crash reporting any more, feel free to remove `opt_out_crash_reporting` from your Fastfile")
+        UI.message("fastlane doesn't have crash reporting anymore, feel free to remove `opt_out_crash_reporting` from your Fastfile")
```

Comment:
The noun form compounds.

Note that for this, I'm using a rule that explicitly looks for `any more[.,]` as that is a dead giveaway. 

There are some cases of `any more$` and `any more[)]` which probably also should be fixed, but the rule is designed to only catch ones that are definitely wrong and not cause me to spend extra hours thinking about other cases. `any more` before a new line could be part of a word wrapped sentence. There are definitely instances of `any more` that are correctly written with two words (but not here...).